example/debug/odp_debug.c

```diff
@@ -378,16 +379,19 @@ static int timer_debug(void)
 		ODPH_ERR("Timer capa failed\n");
 		return -1;
 	}
+	memset(&timer_res_capa, 0, sizeof(odp_timer_res_capability_t));
+	timer_res_capa.res_ns = res;
+	if (odp_timer_res_capability(ODP_CLOCK_CPU, &timer_res_capa)) {
+		ODPH_ERR("Timer resolution capability failed\n");
+		return -1;
+	}
 
-	if (timer_capa.max_tmo.max_tmo < max_tmo)
-		max_tmo = timer_capa.max_tmo.max_tmo;
```

Comment:
Actually, we don't care about resolution, but maximum tmo being long enough that we can print debug info before timer expires. So, this (max_tmo=1sec or max_tmo=capa.max_tmo) should be there first, and then get resolution and min_tmo with odp_timer_res_capability().

Reply:
Ack, I will send the next version.

Reply:
Max tmo capa may be less than 1sec, so we need to first check that. After that res_capa will succeed and we'll use res and min tmo from its output.

```
if (timer_capa.max_tmo.max_tmo < max_tmo)
	max_tmo = timer_capa.max_tmo.max_tmo;

memset(&timer_res_capa, 0, sizeof(odp_timer_res_capability_t));
timer_res_capa.max_tmo = max_tmo;
if (odp_timer_res_capability(ODP_CLOCK_CPU, &timer_res_capa)) {
	ODPH_ERR("Timer resolution capability failed\n");
	return -1;
}
```